README.md

```diff
@@ -70,6 +70,9 @@ The bridge provides a plugin (child) the ability to implement the following APIs
 
 ```
 window.pluginBridge.register({
+  customize: (customization) => {
+    // object of plugin specific configuration values
```

Comment:
I'm interested in defining what that looks like here. 

Though some of the other apis are lacking similar documentation, they benefit from receiving defined data structures in griffon, like events and plugins. We should flush out what that payload looks like and how plugins are expected to use specific values vs global ones.